src/api/githubData/index.tsx

```diff
@@ -0,0 +1,26 @@
+import axios from "axios";
+
+export const getData = async (searchValue: string, sort: string) => {
+  let searchString =
+    "https://api.github.com/search/repositories?q=" + searchValue;
+
+  if (!sort) {
+    searchString = searchString + "&sort=" + sort;
+  }
+
+  const { items = [] } = (await axios.get(searchString)).data || {};
+
+  const filterData = items.map((repository: any) => {
```

Comment:
```suggestion
  const filterData = items.map((repository) => {
```
Any won't be required after defining type